web/src/components/atoms/Icon/index.tsx

```diff
@@ -1,5 +1,6 @@
-import { HomeOutlined } from "@ant-design/icons";
+import { HomeOutlined, DownloadOutlined } from "@ant-design/icons";
 
 export default {
   home: HomeOutlined,
+  download: DownloadOutlined,
```

Comment:
Thanks for changing this. But sorry I think this MIGHT change in another issue sometime later. @Basel-Issmail and I are chatting about it.